app/services/utils/pdf_generator.rb

```diff
@@ -17,15 +17,15 @@ def call
     end
 
     def render_html
-      Slim::Template.new { template_file }.render(context)
+      Slim::Template.new(template_file).render(context)
```

Comment:
We pass the file name as a string in the argument and it reads the file itself instead of using the block and reading the files in the `template_file` method below.